src/app.controller.ts

```diff
@@ -5,13 +5,25 @@ import { AppService } from './app.service';
 export class AppController {
   constructor(private readonly appService: AppService) {}
 
+  // TODO: Deprecate
   @Get('check/:tokenId')
   findOne(@Param('tokenId') tokenId: string) {
     return this.appService.checkAzuki(parseInt(tokenId));
   }
 
+  @Get('v1/check/:tokenId')
+  findOneAzuki(@Param('tokenId') tokenId: string) {
+    return this.appService.getAzukiDetails(parseInt(tokenId));
+  }
+
+  // TODO: Deprecate
   @Get('unclaimed')
   getUnclaimed() {
     return this.appService.getUnclaimed();
   }
```

Comment:
Keeping these old endpoints for now for backward compatibility. Will remove in later PR

Reply:
Versioning is perfect thank you!